src/main/java/edu/kis/powp/jobs2d/command/TransformerCommandVisitor.java

```diff
@@ -0,0 +1,29 @@
+package edu.kis.powp.jobs2d.command;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class TransformerCommandVisitor implements TransformerCommandVisitorInterface{
+
+    private final List<DriverCommand> driverCommandList = new ArrayList<>();
+    private TransformStrategyInterface strategy;
+
+    public TransformerCommandVisitor(TransformStrategyInterface strategy) {
+        this.strategy = strategy;
+    }
+
+    public void run(OperateToCommand operateToCommand) {
+        driverCommandList.add(strategy.execute(operateToCommand));
+    }
+
+    public void run(SetPositionCommand setPositionCommand) {
+        driverCommandList.add(strategy.execute(setPositionCommand));
+    }
```

Comment:
instead of run methods you should implement visit methods from DriverCommandVisitor interface